redstone/client.py

```diff
@@ -896,6 +927,51 @@ def set_instance_allowed_ip_policy(
         ]
         return self._set_policy(resources_list)
 
+    def set_instance_key_create_import_access_policy(
+        self,
+        key_create_import_access_enable: bool,
```

Comment:
I don't understand what this parameter is from its name... please add a comment in the docstring or change its name